mhs/common/mhs_common/workflow/common.py

```diff
@@ -23,11 +23,14 @@ class CommonWorkflow(abc.ABC):
         pass
 
     @abc.abstractmethod
-    async def handle_inbound_message(self, work_description: wd.WorkDescription, payload: str):
+    async def handle_inbound_message(self, message_id: str, correlation_id: str, work_description: wd.WorkDescription,
+                                     payload: str):
         """
         Handles an inbound message from an external system (typically from spine)
-        :param work_description:
-        :param payload:
-        :return:
+
+        :param message_id: ID of the message
```

Comment:
It looks like you actually expect this to be the ID of the message the inbound message is in reference to (which I think makes sense if that's the ID you're using to identify the response on the queue with). This could be made clearer in the parameter name/documentation.